package.json

```diff
@@ -2,6 +2,7 @@
   "scripts": {
     "bootstrap": "lerna bootstrap --force-local",
     "clean": "lerna run clean --stream",
+    "cleanTestData": "lerna run cleanTestData --stream",
```

Comment:
Small Nit: the scripts are inconsistent in their naming convention.  There are scripts with: camel case, snake case, and gulp style.